spec/Section 3 -- Type System.md

```diff
@@ -1800,19 +1849,20 @@ a non-null input type as invalid.
 
 **Type Validation**
 
-1. A Non-Null type must not wrap another Non-Null type.
+1. A *Non-Null type* must not wrap another *Non-Null type*.
 
 
 ### Combining List and Non-Null
 
-The List and Non-Null wrapping types can compose, representing more complex
-types. The rules for result coercion and input coercion of Lists and Non-Null
+The *List type* and *Non-Null types* can compose, representing more complex
```

Comment:
You've been careful to always use the singular elsewhere; I'm not sure if this relates to how the glossary feature in spec-md works. If this need not be singular on these technical grounds then plural is fine :+1: 

```suggestion
The *List type* and *Non-Null type* can compose, representing more complex
```